section3/exercises/arrays.js

```diff
@@ -107,10 +118,25 @@ var chargerType = "car";
 
 // Write your conditional here
 
+//We are checking to see if the user's phone battery life is greater than or equal to 15%
```

Comment:
Is there a way you could modify this so that you only have one line of code with `console.log("You can call an Uber!");` ie - no redundant code?


Just a push - not required for you to re-submit 😊